### ISSUE TITLE
Terminology: Section or Page?

### ISSUE THREAD

**Issue by nikolas** [ccnmtl/django-pagetree#52]
What is the reasoning for using the class name 'Section' for nodes in the pagetree? I think it would make more sense to call them Pages.

**Comment by thraxil** [comment]
Yeah, it's an old debate. I feel like "page" is such an overused and vague term that I kind of wanted to get away from it. I also thought it seemed weird since there is a recursive relationship between them. "Section" is pretty vague and overused as well but at least it made more sense for a section to contain other sections rather than pages containing pages. So, ultimately, I'm not crazy about it either but "section" just seemed like the least bad term to use.

**Comment by nikolas** [comment]
I see. Maybe just something like "Node" would be the most straightforward? It's not a big deal I was just curious. When I first looked at the pagetree code, it wasn't immediately obvious that a "Section" is the primary construct we're dealing with.. it sounds like it's referring to a group of things.

If we did want to change `Section` to something else, I wonder if it would be possible to maintain backwards-compatibility by having a class "alias" or something.

**Comment by nikolas** [comment]
No reason to change this really. This library is still useful, and section is fine.
